site/public/css/table.css

```diff
@@ -82,6 +82,14 @@
     }
 }
 
+
+.mobile-table th,
+.mobile-table td {
+    text-align: center;
+    padding: 5px 10px;
+}
```

Comment:
Is this change necessary here?  As far as I can tell, it will have a wide-ranging effect on tables across the Submitty codebase, and the padding/alignment changes may have unintended consequences.